### PR TITLE
Add pm.max_requests and status/ping path configurations

### PR DESCRIPTION
**Pull request by janbuecker**

### CODE REVIEW

File: fpm/Dockerfile [hunk 1]
```diff
@@ -43,6 +43,9 @@ ENV APP_ENV=prod \
     FPM_PM_START_SERVERS=2 \
     FPM_PM_MIN_SPARE_SERVERS=1 \
     FPM_PM_MAX_SPARE_SERVERS=3 \
+    FPM_PM_MAX_REQUESTS=0 \
+    FPM_PM_STATUS_PATH=/status \
```
Comment (shyim) [inline fpm/Dockerfile:47]: should be prefix it somehow before something like `/.fpm/status`. to not hit any maybe used shopware routes?

Comment (janbuecker) [reply fpm/Dockerfile:47]: I think using a path starting with a dot `.` might has a bad taste. What about:

- `/-/fpm-status`
- `/-/fpm/status`

And continue the same pattern with ping like `/-/fpm/ping`.

Comment (shyim) [reply fpm/Dockerfile:47]: sounds good

Comment (janbuecker) [reply fpm/Dockerfile]: Updated

**Review by shyim — APPROVED** [review]

**Pull request merged**